Changes

```diff
@@ -1,6 +1,27 @@
 Release history of Zonemaster
 
 
+v2023.2.1 2024-03-28
+
+ [Release information]
+ - Fixes errors in release v2023.2
+
+ [Fixes]
+ - Fixes bugs in database migration script in Zonemaster-Backend.
+ - Corrects database upgrade instructions (#1244)
+
+ [Zonemaster product]
+ - This version of Zonemaster also consists of the following components. For
+   each component, see its Changes file or Github release notes for complete
+   release information.
+
+   Zonemaster-LDNS      v4.0.0  https://github.com/zonemaster/zonemaster-ldns/releases/tag/v4.0.0
```

Comment:
Shouldn’t that be bumped to 4.0.1?

Reply:
Yes, missed that. That's why reviewing is good. :-)

Reply:
Fixed.